master/internal/kubernetes/queue.go

```diff
@@ -0,0 +1,215 @@
+package kubernetes
```

Comment:
nonblocking: probably name this file to be `request_queue.go`

Reply:
done